src/auth0-session/utils/cookies.ts

```diff
@@ -22,5 +22,17 @@ export const set = (res: ServerResponse, name: string, value: string, options: C
 };
 
 export const clear = (res: ServerResponse, name: string, options: CookieSerializeOptions = {}): void => {
-  set(res, name, '', { ...options, maxAge: 0 });
+  const { domain, path, secure, sameSite } = options;
```

Comment:
sameSite is not being passed to `set`, because it's never added to `clearOptions`.

Reply:
Ooops didn't see you had already closed the PR.

Reply:
That's intentional - you only need to pass `path` and `domain` (and sometime `secure`) when clearing the cookie, I'm just closing this because I can't reproduce #569 on https://localhost and I'm doing a bit more investigation